### DISCUSSION TITLE
Ability to hide "No Package Published Yet" when not logged

### DISCUSSION THREAD

**Discussion by ioideanh** [verdaccio/verdaccio#2468] [Ideas]
We have a number of packages in Verdaccio, all packages are private and require you to be authenticated to see them.

Issue is the landing page shows the "No Package Published Yet" home screen, would like to disabled this.

Along the same lines the "i" info icon should be hidden until a user is logged in.

thanks

**Comment by juanpicado** [comment]
Some ideas of implementation:

- Add new flag under web property to disable this banner (Easy)
- Send a flag from the backend via VERDACCIO_UI_OPTIONS checking if there is already some package published (Medium) <-- I see much work for so low value

More ideas are welcome
